scripts/setup

```diff
@@ -75,12 +75,19 @@ STOP_EXECUTION=true
 CREATE_LAZY=""
 DONT_STOP=""
 PREFIX=""
+ENABLE_WARNINGS=false
```

Comment:
The warnings should be active by default.

Reply:
I change the behaviour as you expect. Have a look!